ikasaneip/developer/mvn-archetype/ikasan-standalone-db-jms-im/src/main/resources/archetype-resources/jar/src/main/resources/application.properties

```diff
@@ -22,9 +22,18 @@ spring.liquibase.change-log=classpath:db-changelog-master.xml
 spring.liquibase.enabled=true
 
 # health probs and remote management (optional)
-management.endpoints.web.expose=*
-management.server.servlet.context-path=/manage
+management.endpoints.enabled-by-default=false
+management.endpoint.info.enabled=true
+management.endpoint.health.enabled=true
+management.endpoint.logfile.enabled=true
+management.endpoints.web.exposure.include=info,health,logfile
 management.endpoint.shutdown.enabled=true
+#management.endpoints.web.base-path=/rest
```

Comment:
@andrzej-majewski i'll approve but do we need to leave commented out items in the files?